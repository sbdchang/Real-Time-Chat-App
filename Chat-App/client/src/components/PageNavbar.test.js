import React from 'react';
import ReactDOM from 'react-dom';
import { render, screen } from '@testing-library/react';
import PageNavbar from './PageNavbar';
import renderer from 'react-test-renderer';

it('PageNavBar Snapshot  ', () => {
    const tree = renderer.create(<PageNavbar />).toJSON();
    expect(tree).toMatchSnapshot();
  });

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<PageNavbar />, div);
});

/*
it('renders contacts', () => {
  render(<Mainview />);
  expect(screen.getByText('Contacts')).toBeInTheDocument();
});

*/
