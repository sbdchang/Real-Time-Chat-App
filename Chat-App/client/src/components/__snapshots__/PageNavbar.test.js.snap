// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNavBar Snapshot   1`] = `
<div
  className="PageNavbar"
>
  <nav
    className="navbar navbar-expand-lg navbar-light bg-light"
  >
    <span
      className="navbar-brand center"
    >
      CIS557
    </span>
    <div
      className="collapse navbar-collapse"
      id="navbarNavAltMarkup"
    >
      <div
        className="navbar-nav"
      />
    </div>
  </nav>
</div>
`;
